predicate/predicate.py

```diff
@@ -226,3 +263,28 @@ def values(self, obj):
                 node.children[lookup] = value
             results.append(node)
         return results
+
+
+def get_values_list(obj, *lookups, **kwargs):
+    """
+    Convenience method that simulates the effect of QuerySet.values_list.
+
+    Other than ordering, this should behave identically to the following for
+    Django model instances and lookups on django fields:
+        SomeModel.objects.filter(pk=obj.pk).values_list(*lookups, **kwargs)
+    """
+    flat = kwargs.pop('flat', False)
+    if kwargs:
+        raise TypeError('Unexpected keyword arguments to values_list: %s' %
+                        (list(kwargs),))
```

Comment:
That was copied from `QuerySet.values_list()`. I think the most idiomatic way to express it would be `kwargs.keys()`.  Fixed in e30f552f8c7b0c9f6241c0ba304b595fab882665.